pymeasure/instruments/agilent/agilent33500.py

```diff
@@ -48,6 +46,317 @@
 # string_validator = joined_validators(capitalize_string, strict_discrete_set)
 
 
+class Agilent33500Channel(Channel):
+    """Implementation of a base Agilent 33500 channel"""
+
+    def __init__(self, instrument, id):
+        super().__init__(instrument, id)
+
+    shape = Instrument.control(
+        "SOUR{ch}:FUNC?",
+        "SOUR{ch}:FUNC %s",
+        """ A string property that controls the output waveform. Can be set to:
+        SIN<USOID>, SQU<ARE>, TRI<ANGLE>, RAMP, PULS<E>, PRBS,  NOIS<E>, ARB, DC. """,
+        validator=strict_discrete_set,
+        values=["SIN", "SQU", "TRI", "RAMP", "PULS", "PRBS", "NOIS", "ARB", "DC"],
+    )
+
+    frequency = Instrument.control(
+        "SOUR{ch}:FREQ?",
+        "SOUR{ch}:FREQ %f",
+        """ A floating point property that controls the frequency of the output
+        waveform in Hz, from 1 uHz to 120 MHz (maximum range, can be lower depending
+        on your device), depending on the specified function. Can be set. """,
```

Comment:
```suggestion
        on your device), depending on the specified function.""",
```
I think "can be set" is implied by the verb control.
Similar comments for other places.

Reply:
Fair. Removed a bunch of "can be set" in https://github.com/pymeasure/pymeasure/pull/763/commits/3e71c1eee00e7ff48596b6418855cf6a67fe831e